src/index.ts

```diff
@@ -53,20 +35,35 @@ const resolvers = {
   },
 
   Mutation: {
-    createUser: async (_, { data }) => {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    createUser: async (parent, args, context, info) => {
```

Comment:
A variáveis q vc n usa vc pode substituir por `_` (mais de um caso seja mais de uma variável). Assim vc pode tirar o disable do lint.

Reply:
Tentei fazer isso mas o TS reclamou que eu estava utilizando o identificador duas vezes

Reply:
Mesmo você usando `_` e `__`, por exemplo? (1 e 2 undescores)?

Reply:
`_` e `__` funcionam para remover o problema de não estar sendo usado, mas ele ainda reclama que está com o tipo 'any' implicito

`Parameter '__' implicitly has an 'any' type, but a better type may be inferred from usage.`